packages/documentation/src/stories/components/forms/input/input.stories.ts

```diff
@@ -160,7 +160,7 @@ function render(args: Args, context: StoryContext) {
   const contextual: (TemplateResult | null)[] = [
     args.validation === 'is-valid' ? html` <p class="valid-feedback">Ggranda sukceso!</p> ` : null,
     args.validation === 'is-invalid' ? html` <p class="invalid-feedback">Eraro okazis!</p> ` : null,
-    args.hint !== '' ? html` <div class="form-text">${args.hint}</div> ` : null,
+    args.hint !== '' ? html` <div class="form-hint">${args.hint}</div> ` : null,
```

Comment:
Also connect this on the input element via `aria-describedby`.

```suggestion
    args.hint !== '' ? html` <p class="form-hint" id="form-hint-example">${args.hint}</p> ` : null,
```